### DISCUSSION TITLE
Does Lightkurve smooth-out or minimize intrinsic variability?

### DISCUSSION THREAD

**Discussion by Brad-Schaefer** [lightkurve/lightkurve#1252] [Q&A]
Does Lightkurve smooth-out, minimize, or remove instrinsic variability in the light curve of a target star in TESS light curves?  I am asking because my Lightkurve extracted light curves for four TESS sectors all have the target showing only small amplitude chaotic variability with a time scale of one to ten days with full peak-to-peak amplitudes of 0.004 mag.  This is a trouble because I have four other unimpeachable light curve sources that show the same target to have chaotic variations on the 1-10 day timescale -- but with full amplitudes of 0.11--0.20 mag.  It is very unlikely that the target knew to turn off most of its variability just for the four TESS sectors and then switch back on for my other data sources.  Two of my alternative data sources are contemporaneous with the TESS sectors, and they have similar spectral sensitivity, so we have no explanation from the amplitude due to long-term change in the variability or to differences due to color.  With four independent data sources giving 0.11--0.20 mag amplitude variability, yet the LightKurve light curve giving only 0.004 mag amplitude, I have to be concerned that something is wrong with the Lightkurve photometry.  Further, the SPOC light curve for sector 52 shows variability at the 0.2 mag level, while the Lightkurve product shows only Poisson noise below the 0.004 mag level for the exact same input data.

OK, let me give some details that people can use for consideration.  My target is IRAS 00500+6713, which is the remnant left behind by the supernova seen by Chinese and Japanese observers back in the year 1181 AD.  My science question is to find a coherent periodic modulation from ~40 seconds up to ~10 days, or to place an upper limit on any such periodic modulation.  (It turns out that a severe limit on any periodic modulation from an orbital period is the last and biggest constraint that eliminates one of the two competing models, and this solves the case.  Or is there really an orbital period in the TESS data that has been minimized by Lightkurve?  So my final science conclusion hangs on whether there is a strict limit on any orbital modulation as seen by TESS.)  This target is a roughly 15th mag isolated star.  With MAST, the target position is about two arc-seconds off of the central star's coordinates.  The J2000 coordinates of the target star is 00:53:11.21  +67:30:02.4.  The target was well measured with TESS sectors 18, 24, 25, and 52.  The first three sectors only have FFI data at 1800s time resolution.  Sector 52 has 20 TPFs, while MAST contains SPOC light curves with 20-second and 120-second time resolutions.  I have run Lightkurve for the FFI data, for the PLD, PLD2, and RC correctors.  Here is the plot of sector #18 from Lightkurve:
      
<img width="378" alt="image" src="https://user-images.githubusercontent.com/113924103/191072749-2d106856-6154-4cee-8507-dbafffa95fe0.png">

The point of this plot is to see that the variability returned is ~0.1% for this sector.  To see what my photometry aperture is, here is the plot:
 
<img width="283" alt="image" src="https://user-images.githubusercontent.com/113924103/191072817-91a03d60-5fe5-41eb-a489-482adc9a648e.png">

I have also run the same code with a shifted 3x3 box to a background area, and the average PDCSAP flux is 1101.  The PDCSAP flux from the target with the RC corrector is 2660 with no background subtraction.  

	Here is the same info for sector 52:
 
<img width="261" alt="image" src="https://user-images.githubusercontent.com/113924103/191072873-dd66fd9f-e11e-4c3d-865d-6c9a6b3e0108.png">

<img width="468" alt="image" src="https://user-images.githubusercontent.com/113924103/191072906-ddbacab1-dc5d-44a3-8b4e-9bb5dd82e678.png">
 
Again, we see that Lightkurve gives only a very small variability (consistent with the Poisson noise) of up to 0.4%.  (Again, the question in hand is whether this variability in Lightkurve has somehow removed much of the intrinsic variability?)  The SPOC light curve for sector 52 shows a large amount of noise with 20-seconds time resolution, but when binned up to 1800 seconds, the light curve shows variations of 0.2 mag.  This directly contradicts the amplitude of variability from Lightkurve for the exact same data.  So here we have a strong pointer that Lightkurve is somehow taking out intrinsic variability in the light curves it produces.

	So what is going on with the amplitude of variability in the Lightkurve output?  Has the intrinsic variability been somehow supressed?  If so, what are the details of the supression and what types of intrinsic variability are taken out?  Please help me with understanding what is going on here?

**Comment by bibinthomas123** [comment]
yes, some of the processing can suppress intrinsic variability, but Lightkurve's basic aperture photometry should not do this by itself.

The important distinction is between the raw/SAP extraction and the detrended products. PDCSAP/CBV corrections are specifically designed to remove systematic trends, and long-period astrophysical variability can be attenuated in the process. Lightkurve itself notes that PDC processing can remove variability on roughly ≥10-day timescales.

For PLD, it's a little more nuanced: Lightkurve includes a spline component intended to model the target's intrinsic long-term variability and has restore_trend=True by default. However, PLD is still a regression model and can overfit if the setup is not appropriate.

So in your case, 0.004 mag versus 0.11–0.20 mag is much too large to dismiss as normal dilution. I would first compare:

raw aperture/SAP → PLD → PDCSAP

on the exact same Sector 52 data. If the 0.2-mag signal exists in SAP but disappears after PLD/PDCSAP, then you've identified the problem.

And because you're specifically searching for 1–10 day variability, I would not use the corrected Lightkurve curve to claim a stringent upper limit until you test the correction with injected signals at those periods. That will tell you directly whether the pipeline is suppressing the signal you're looking for.
